runtime/port/unix/j9sysinfo.c

```diff
@@ -250,6 +250,10 @@ static intptr_t getAIXPPCDescription(struct J9PortLibrary *portLibrary, J9Proces
 #if !defined(__power_9)
 #define POWER_9 0x20000 /* Power 9 class CPU */
 #define __power_9() (_system_configuration.implementation == POWER_9)
+#if !defined(J9OS_I5_V6R1)
+#define PPI9 0x4E
+#define __phy_proc_imp_9() (_system_configuration.phys_implementation == PPI9)
+#endif /* !defined(J9OS_I5_V6R1) */
```

Comment:
These definitions are IBM i specific but the code isn't ifdef'd for that platform.  Shouldn't this be ifdef'd something like:
```
#if defined(J9OS_I5) && !defined(J9OS_I5_V6R1)
```